pages/about/code-of-conduct.vue

```diff
@@ -258,6 +258,12 @@ export default {
         @apply mt-20;
     }
 }
+@media (max-width: 500px) {
+    .img_text {
+        font-size: 0.4rem;
+        white-space: normal;
```

Comment:
Please use Tailwind syntax with `@apply` for these styles.

Reply:
OK